graphql/integration_spec.json

```diff
@@ -1954,6 +2074,59 @@
           }
         }
       }
+    },
+    {
+      "query_file": "test-files-query-display.graphql",
+      "result": {
+        "data": {
+          "testFiles": [
+            {
+              "name": "exec 1 display!!",
+              "files": [
+                {
+                  "name": "awesome_test!!!",
+                  "link": "www.wikipedia.com",
+                  "visibility": "the whole world"
+                }
+              ]
+            },
+            {
+              "name": "exec 2 display!!",
+              "files": [
+                {
+                  "name": "THIS IS REALLY COOL",
+                  "link": "www.wikipedia.com",
+                  "visibility": "the whole world"
+                },
+                {
+                  "name": "really cool test file",
+                  "link": "www.wikipedia.com",
+                  "visibility": "the whole world"
+                }
+              ]
+            }
+          ]
+        }
+      }
+    },
+    {
+      "query_file": "test-files-query-exec.graphql",
+      "result": {
+        "data": {
+          "testFiles": [
+            {
+              "name": "exec 1 display!!",
+              "files": [
+                {
+                  "name": "awesome_test!!!",
+                  "link": "www.wikipedia.com",
+                  "visibility": "the whole world"
+                }
+              ]
+            }
```

Comment:
`name` in outer object and `name` in nested object is a little ambiguous / makes it hard for me to understand what the names refer to